department-of-veterans-affairs/abd-vro
# Rewrite lookup table files to improve readability

## What was the problem?

 - This PR rewrites the lookup table for the Contention Classification to use a single .csv file per request of product management.
 - This PR updates the lookup tables classes in lookup_table.py to return a dict containing keys `"classification_code"` and `"classification_name"` in order to get access to both pieces of information at the same time

Associated tickets or Slack threads:
- https://github.com/department-of-veterans-affairs/vagov-claim-classification/issues/476

## How does this fix it?[^1]

 - Design provided a new format for a .csv file. This PR uses ContentionTextLookupTable to parse the new .csv
 - 

## How to test this PR
- manual testing of `/classifier` and `/vagov-claim-classifier`
- run `pytest`


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
